### ISSUE TITLE
(Interactive Description) The scale for charge movement in wall is not correct

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#490]
When balloon has "**a few** more negative charges than positive charges", negative charges in wall are being described as "move away from Yellow Balloon **a lot**."

"a few more negative charges" should only cause a small induced charge effect, so the wall charge movement should be in the first region above no movement which is "a little bit". The region fro several is also off.

Here's the balloon's charge amount scale
- {{no more}} negative charges than positive charges
- {{a few more}}
- {{several more}}
- {{many more}}

Here's the movement of charges in wall scale
- In {{wall-position}} wall, {{no change in charges}}.
- Negative charges in {{wall-position}} wall move way from Yellow Balloon {{a little bit}}.
- Negative charges in {{wall-position}} wall move way from Yellow Balloon {{a lot}}.
- Negative charges in {{wall-position}} wall move way from Yellow Balloon {{quite a lot}}.

See screen shot
![Screen Shot 2021-04-01 at 10 14 46 AM](https://user-images.githubusercontent.com/1902116/113295856-4b0f9800-92d3-11eb-8ba9-c327010ea910.png)

To resolve, we may need to adjust both the regions describing amount of negative charges on balloon and movement in wall.
@jessegreenberg, when you have a chance, could you provide the current ranges for the regions being used right now for both balloon charge amount and charge movement amount. That should help me.

NOTE - when two balloons are present, there is a combined induced charge effect, so we need to be careful of that situations, as well. For example, when "Negative charges in wall move away from **balloons** a little bit/ a lot/ quite a lot".

**Comment by terracoda** [comment]
Tagging @emily-phet and @jbphet so at next meeting we can discuss if this should be a blocking issue.

**Comment by jessegreenberg** [comment]
Can easily reproduce.  I don't think this is a sim bug but the way the description for induced charge has been working for a while. The fix will be to modify the ranges.

Here is the map that gets the strings:
```js
const INDUCED_CHARGE_DESCRIPTION_MAP = {
  A_LITTLE_BIT: {
    range: new Range( 0, 10 ),
    description: aLittleBitString
  },
  A_LOT: {
    range: new Range( 10, 20 ),
    description: aLotString
  },
  QUITE_A_LOT: {
    range: new Range( 20, Number.MAX_VALUE ),
    description: quiteALotString
  }
};
```

The displacement value in @terracoda's screenshot is around 14, producing the "a lot" string.

**Comment by jessegreenberg** [comment]
I just updated to this range:

```js
const INDUCED_CHARGE_DESCRIPTION_MAP = {
  A_LITTLE_BIT: {
    range: new Range( 0, 20 ),
    description: aLittleBitString
  },
  A_LOT: {
    range: new Range( 20, 35 ),
    description: aLotString
  },
  QUITE_A_LOT: {
    range: new Range( 35, Number.MAX_VALUE ),
    description: quiteALotString
  }
};
```

and it broke many unit tests, they will have to be updated with the new expectations.

**Comment by jessegreenberg** [comment]
Done in the above commit. @terracoda could you please review and verify it is behaving better now?

**Comment by terracoda** [comment]
Ok, I think this scale being for the charge displacement/movement in the wall is now correct, so closing this issue.

Unfortunately, I found three of other issues for which I will open new issues:
1. sometimes I hear "article web application" when putting my focus on a balloon
2. when balloons are both released at next to each other at wall, I get a single description of the balloon, rather than a combined one.
3. In the Preferences menu, I see black focus highlights for reading blocks when using keyboard navigation with or without Interactive Highlights
